src/imgui/Shortcuts.hh

```diff
@@ -0,0 +1,79 @@
+#ifndef IMGUI_SHORTCUTS_HH
+#define IMGUI_SHORTCUTS_HH
+
+#include "ImGuiUtils.hh"
+
+#include "strCat.hh"
+#include "imgui_stdlib.h"
+
+#include <functional>
+#include <optional>
+#include <string_view>
+#include <vector>
+
+namespace openmsx {
+
+class ImGuiManager;
+class GlobalSettings;
+class SettingsConfig;
+
+enum ShortcutIndex {
+	GOTO_MEM = 0,
+	STEP,
+	BREAK,
+	GOTO_DISASM,
+	MOVE_UP_MEM,
+	MOVE_DOWN_MEM,
+	MOVE_LEFT_MEM,
+	MOVE_RIGHT_MEM,
+	NUM_SHORTCUTS,
+};
+
+class Shortcuts final
```

Comment:
Detail:
This `final` annotation is not wrong, and I can understand why it could be useful (it prevents other code from inheriting from `Shortcuts`).
Though in c++ (in general, so not only in openMSX) the habit is to only use `final` on leaf classes in a class-hierarchy. So omit this annotation on classes that are not meant to be used in a class-hierarchy.
But this is only a convention, and conventions can evolve over time.